data/Language/JSON.yaml

```diff
@@ -0,0 +1,15 @@
+identifier: JSON
+wikidata: Q2063
+full_name: JavaScript Object Notation
+extensions: 
+  - json
+line_continuation: ''
+delimiters:
+  # From coala JSON language definition
+  # JSON don't have any comments
+  # String delimiter:
+  - double_quote
```

Comment:
JSON string have an escape character. Choose a more specific delimiter

Reply:
Oh, that's right. I was a bit confused by what delimiter to use, but I'll check them

Reply:
Yup, JSON uses backslash escaping.